flexget/plugins/clients/flood.py

```diff
@@ -0,0 +1,131 @@
+      destination: string().optional(),
+      // Tags
+      tags: array(string().regex(noComma, TAG_NO_COMMA_MESSAGE)).optional(),
+      // Whether destination is the base path [default: false]
+      isBasePath: boolean().optional(),
+      // Whether destination contains completed contents [default: false]
+      isCompleted: boolean().optional(),
+      // Whether contents of a torrent should be downloaded sequentially [default: false]
+      isSequential: boolean().optional(),
+      // Whether to use initial seeding mode [default: false]
+      isInitialSeeding: boolean().optional(),
+      // Whether to start torrent [default: false]
+      start: boolean().optional(),
+    });"""
+    def add_torrent_urls(self, config, urls):
+        if not self.connected:
+            raise plugin.PluginError('Not connected.')
+
+        response = self._request('post', '{}:{}/api/torrents/add-urls'.format(config['host'], config['port']), json={
+            'urls': urls,
+            'destination': config['directory'],
+            'tags': config['tags'],
+            'start': True
+        })
+
+        if response.status_code == 200:
+            logger.debug('Successfully added torrent to Flood')
+        else:
+            raise plugin.PluginError('Failed to add torrent to Flood. Error {}'.format(response.status_code))
+
+    """// POST /api/torrents/add-files
+    export const addTorrentByFileSchema = object({
+    // Torrent files in base64
+    files: array(string()).nonempty(),
+    // Path of destination
+    destination: string().optional(),
+    // Tags
+    tags: array(string().regex(noComma, TAG_NO_COMMA_MESSAGE)).optional(),
+    // Whether destination is the base path [default: false]
+    isBasePath: boolean().optional(),
+    // Whether destination contains completed contents [default: false]
+    isCompleted: boolean().optional(),
+    // Whether contents of a torrent should be downloaded sequentially [default: false]
+    isSequential: boolean().optional(),
+    // Whether to use initial seeding mode [default: false]
+    isInitialSeeding: boolean().optional(),
+    // Whether to start torrent [default: false]
+    start: boolean().optional(),
+    });"""
+    def add_torrent_files(self, config, data):
+        if not self.connected:
+            raise error('Not connected')
+
+        response = self._request('post', '{}:{}/api/torrents/add-files'.format(config['host'], config['port']), json=data)
+
+        if response.status_code == 200:
+            logger.debug('Successfully added torrent to Flood')
+        else:
+            raise plugin.PluginError('Failed to add torrent to Flood. Error {}'.format(response.status_code))
+
+    #@plugin.priority(120)
+    #def on_task_download(self, task, config):
+
+    @plugin.priority(135)
+    def on_task_output(self, task, config):
+        if task.accepted and self.authenticate(config):
+            self.add_torrent_urls(config, [entry['url'] for entry in task.accepted])
```

Comment:
It would be better to pass entries, one might want to use some fields from it in the future like "tag".